releasenotes/notes/prelude-release-7.49.0-f22ebb0a2bed17ec.yaml

```diff
@@ -0,0 +1,5 @@
+prelude:
+    |
+    Release on: 2023-11-02
+
+    - Please refer to the `7.49.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7490>`_ for the list of changes on the Core Checks
```

Comment:
```suggestion
    - Refer to the `7.49.0` tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7490> for the list of changes on the core checks.
```

Reply:
This is generated part so I can make changes to the template when it comes to wording. I would like to avoid changing the ` position, as this is just a temporary file which later is used to generate release changelog, which is in rst format and this may break links.

Reply:
Ah sure, I wasn't sure if that was a typo or something else.